blackbox/docs/best_practice/bootstrap_checks.txt

```diff
@@ -0,0 +1,63 @@
+.. highlight:: sh
+
+.. _bootstrap_checks:
+
+================
+Bootstrap Checks
+================
+
+If you are binding to a network reachable IP address, CrateDB performs a number
+of bootstrap checks during startup. These checks examine your setup and will
+prevent startup if a problem is detected.
+
+This best practices guide is intended to help you configure your setup so that
+CrateDB passes the bootstrap checks and can perform optimally.
+
+If you are binding to the loopback address, the bootstrap checks will not be
```

Comment:
I think this is fine? "will not be run" sounds more natural to me than "will not run". the difference is that with "will not run" implies they happen outside of crate (technically true, because they happen with ES, but this isn't the framing we want to use) so "will not be run" implies crate is deciding not to run them

thoughts?